run_network_test.go

```diff
@@ -146,48 +146,147 @@ func TestRunPort(t *testing.T) {
 	hostIP, err := getNonLoopbackIPv4()
 	assert.NilError(t, err)
 	type testCase struct {
-		listenIP  net.IP
-		connectIP net.IP
-		port      int
-		err       string
+		listenIP       net.IP
+		connectIP      net.IP
+		hostPort       string
+		containerPort  string
+		connectURLPort int
+		assert         string
+		err            string
```

Comment:
`assert string` should be probably `runExitCode int` or `runShouldSuccess bool`

Reply:
done